web/admin/src/Admin.js

```diff
@@ -83,19 +82,26 @@ export default class Admin extends PureComponent {
   componentDidUpdate() {
     if (!this.state.sessionId) {
       const sessionIds = Object.keys(this.state.sessions)
-      if (sessionIds.length) this.setState({ sessionId: sessionIds[0] })
+      if (sessionIds.length) {
+        this.setState({ sessionId: sessionIds[0] })
+      }
+    }
+  }
+
+  isSessionEnded = () => {
+    const { publicSessions, sessionId } = this.state
+    if (publicSessions[sessionId]) {
+      const currentSession = publicSessions[sessionId]
+      if (currentSession.state === 'ENDED') {
+        return false
```

Comment:
Need to flip true/false for this function

Reply:
That makes sense with the new function name but then I would need to do: const exportIsDisabled = !this.isSessionEnded()

Reply:
correct